internal/cmd/local/docker/docker.go

```diff
@@ -209,8 +209,7 @@ func (d *Docker) MigrateComposeDB(ctx context.Context, volume string) error {
 	pterm.Debug.Println(fmt.Sprintf("Created initial migration container '%s'", conCopy.ID))
 
 	// docker cp [conCopy.ID]]:/$migratePGDATA/. ~/.airbyte/abctl/data/airbyte-volume-db/pgdata
-	userHome, _ := os.UserHomeDir()
-	dst := filepath.Join(userHome, ".airbyte", "abctl", "data", "airbyte-volume-db", "pgdata")
+	dst := filepath.Join(paths.Data, "airbyte-volume-db", "pgdata")
```

Comment:
I like where this is going, i think we could go even a little further and add some convenience methods like `ensureDir`, `ensureFile`, etc. All of which would be relative to `$HOME/.airbyte`

Reply:
That's the plan, but I'll do that as part of a follow-up PR (after #40 has been merged)